### PR TITLE
Enhan/reeveng

### PR DESCRIPTION
**Pull request by reeveng**
- removed override and other extra stuff from help command
- formatted the way you like

hope you have a better overview this time :wink: :heart:

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-case-declarations */
```
Comment (rozbrajaczpoziomow) [inline index.js]: You can't do that...that's what the rule is enabled for...

File: index.js [hunk 2]
```diff
@@ -122,61 +123,104 @@ Twitch.Client = new tmi.Client({
...
+       	* request help
+       	*
+       	* @returns {string} a list of available commands
+       	*/
+		case 'help':
+			let viewerCommands = [
+				'!help',
+				'!link'
+			];
+			let adminCommands = ['!new', '!start', '!override', '!eval'];
```
Comment (rozbrajaczpoziomow) [inline index.js]: Doesn't respect `Twitch.Config.prefix`

File: index.js [hunk 3]
```diff
@@ -122,61 +123,104 @@ Twitch.Client = new tmi.Client({
...
+
+			if(Twitch.isAdmin(tags.username))
+				return send(
+					`list of commands: ${[...viewerCommands, ...adminCommands].join(
+						', '
+					)}`
+				);
+
```
Comment (rozbrajaczpoziomow) [inline index.js:160]: Join this return send(...) thing into 1 line

File: index.js (hunk 2 above)
Comment (reeveng) [reply index.js]: true 5Head

**Comment by reeveng** [comment]
fixed

**Comment by rozbrajaczpoziomow** [comment]
Closing because of https://github.com/rozbrajaczpoziomow/coc-bot-electric-boogaloo/commit/b7a3d7fa47959e489baaa50ad51cab2042a1fac8
Thanks for the idea!

**Pull request closed without merging**